extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php

```diff
@@ -111,6 +111,10 @@ public function getMenuNodes() {
 			[3] // max 3 links
 		);
 
+		if ( $wgLang->getCode() !== 'en' ) {
```

Comment:
You mean like this:

`		if ( $wgLang->getCode() === 'en' ) {
			$hubsNodes = (new NavigationModel(true /* useSharedMemcKey */))->getTree(
				NavigationModel::TYPE_MESSAGE,
				'global-navigation-menu-hubs',
				[3] // max 3 links
			);
		} else {
			$hubsNodes = [];
		}
`

The reason I went with overriding for non-English as opposed to the way I pasted above is that other code in the file seems to prefer that method:

I see `if ( $lang !== 'en' )` and if ( $lang !== self::DEFAULT_LANG)` other places in the file.

I'm fine with changing it, so please let me know if I should choose the newly pasted code or stick with the convention in the file.

PS: Regardless of that, I will change 'en' to self::DEFAULT_LANG, also in getWAMLinkForLang since it is inconsistent there compared to the rest of the file.